index.html

```diff
@@ -1386,7 +1386,7 @@ <h2>Authentication</h2>
 do with that information.
             </p>
             <p>
-This is useful to any <em>authentication verifier</em> that needs to check to
+This is useful to any party verifying the authentication that needs to check to
 see if an entity that is attempting to [=authenticate=] is, in fact,
 presenting a valid proof of authentication. When a <em>verifier</em> receives
```

Comment:
```suggestion
This is useful to any party verifying the authentication — any <em>verifier</em> —
that needs to check whether an entity that is attempting to [=authenticate=] is
presenting a valid proof of authentication. When such a <em>verifier</em> receives
```

Reply:
The point of this change was to eliminate the use of the term _verifier_ with a different meaning than the definition of _verifier_ in VCDM.  See my comment on this topic at https://github.com/w3c/controller-document/issues/48#issuecomment-2321838164.

I'd be good with this change if you delete the " — any <em>verifier</em> —" part.

Reply:
Revising to eliminate *both* uses of the term `verifier` with that meaning. Also replacing `party` the introduction of which I believe the WG reconsidered some time ago.
```suggestion
This is useful to any entity verifying authentication that needs to check
whether an entity that is attempting to [=authenticate=] is presenting a valid
proof of authentication. When such an authentication verifying entity receives
```